src/org/omegat/core/data/TMXEntry.java

```diff
@@ -171,20 +175,48 @@ public boolean equalsTranslation(TMXEntry other) {
         }
         return true;
     }
-    
+
+    /**
+     * We only hold origin property in ProjectTMXEntry.
```

Comment:
Only one small detail: this comment still refers to ProjectTMXEntry which is not anymore in the pull request